boxoffice/views/utils.py

```diff
@@ -76,14 +76,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def csv_response(headers, rows):
+def csv_response(headers, rows, row_handler=None):
     """
     Returns a response, with mimetype set to text/csv,
     given a list of headers and a two-dimensional list of rows
+
+    Accepts an optional row_handler function that can be used to transform the row. The row
+    must be a list or a tuple of values.
     """
     stream = StringIO()
     csv_writer = unicodecsv.writer(stream)
     csv_writer.writerow(headers)
     for row in rows:
-        csv_writer.writerow(row)
+        if callable(row_handler):
+            csv_writer.writerow(row_handler(row))
+        else:
+            csv_writer.writerow(row)
```

Comment:
Depending on `callable`'s calling cost, you may want to put it outside the `for` loop, with two loops. If it can stay inside, this is more concise: `csv_writer.writerow(row_handler(row) if callable(row_handler) else row)`. I think there's a `.writerows(…)` as well so you can eliminate the `for` loop.